README.md

```diff
@@ -139,6 +172,7 @@ License & Authors
 - Author:: Aaron Kalin (<aaron.kalin@dnsimple.com>)
 - Author:: Jacobo García (<jacobo.garcia@dnsimple.com>)
 - Author:: Anthony Eden (<anthony.eden@dnsimple.com>)
+- Author:: Stephan Kemper (<stephan.kemper@viasat.com>)
```

Comment:
Can you remove this line? Happy to list you as a contributor for now, if you would like to help with maintenance we can talk about that and get you on there.